cmd/app/unlink.go

```diff
@@ -0,0 +1,137 @@
+			{
+				Meaning: "Remove an existing app from the project",
+				Command: "app unlink",
+			},
+			{
+				Meaning: "Remove a specific app without using prompts",
+				Command: "app unlink --team T0123456789 --app A0123456789 --environment deployed",
+			},
+		}),
+
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			clients.Config.SetFlags(cmd)
+			return cmdutil.IsValidProjectDirectory(clients)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+			clients.IO.PrintTrace(ctx, slacktrace.AppUnlinkStart)
+
+			app, err := UnlinkCommandRunE(ctx, clients, cmd, args)
+			if err != nil {
+				return err
+			}
+			return printUnlinkSuccess(ctx, clients, app)
+		},
+		PostRunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+			clients.IO.PrintTrace(ctx, slacktrace.AppUnlinkSuccess)
+			return nil
+		},
+	}
+	return cmd
+}
+
+// UnlinkCommandRunE executes the unlink command, prints output, and returns any errors.
+func UnlinkCommandRunE(ctx context.Context, clients *shared.ClientFactory, cmd *cobra.Command, args []string) (types.App, error) {
+	// Get the app selection from the flag or prompt
+	selection, err := unlinkAppSelectPromptFunc(ctx, clients, prompts.ShowAllEnvironments, prompts.ShowInstalledAndUninstalledApps)
+	if err != nil {
+		return types.App{}, err
+	}
+
+	// Confirm with user unless --force flag is used
+	if !clients.Config.ForceFlag {
+		proceed, err := confirmUnlink(ctx, clients.IO, selection)
+		if err != nil {
+			return types.App{}, err
+		}
+		if !proceed {
+			clients.IO.PrintInfo(ctx, false, "\n%s", style.Sectionf(style.TextSection{
+				Emoji: "thumbs_up",
+				Text:  "Your app will not be unlinked",
+			}))
+			return types.App{}, nil
+		}
+	}
+
+	// Remove the app from the project
+	app, err := clients.AppClient().Remove(ctx, selection.App)
+	if err != nil {
+		return types.App{}, err
+	}
+
+	// Clean up empty files
+	clients.AppClient().CleanUp()
+
+	return app, nil
+}
+
+// confirmUnlink prompts the user to confirm unlinking the app
+func confirmUnlink(ctx context.Context, IO iostreams.IOStreamer, selection prompts.SelectedApp) (bool, error) {
+	IO.PrintInfo(ctx, false, "\n%s", style.Sectionf(style.TextSection{
+		Emoji: "warning",
+		Text:  "Confirm Unlink",
+		Secondary: []string{
+			fmt.Sprintf("App (%s) will be removed from this project", selection.App.AppID),
+			fmt.Sprintf("Team: %s", selection.Auth.TeamDomain),
+			"The app will not be deleted from Slack",
+			"You can re-link it later with 'slack app link'",
+		},
+	}))
+
+	proceed, err := IO.ConfirmPrompt(ctx, "Are you sure you want to unlink this app?", false)
+	return proceed, err
+}
+
+// printUnlinkSuccess displays success message after unlinking
+func printUnlinkSuccess(ctx context.Context, clients *shared.ClientFactory, app types.App) error {
+	clients.IO.PrintInfo(ctx, false, "\n%s", style.Sectionf(style.TextSection{
+		Emoji: "white_check_mark",
+		Text:  "App Unlinked",
```

Comment:
```suggestion
		Emoji: "unlock",
		Text:  "App Unlink",
```

🏷️ suggestion: Similar suggestion to the outputs above!